### DISCUSSION TITLE
Please keep supporting Windows 8.1

### DISCUSSION THREAD

**Discussion by ghost** [git-for-windows/git#4975] [General]
The MSYS2 installer version msys2-x86_64-20240113.exe is still supporting Windows 8.1. I'm running this version and have updated to latest packages (including the msys runtime) without any problems.

**Comment by rimrul** [comment]
There are no current plans to drop Windows 8.1 support AFAIK.

**Reply by ghost** [reply]
> Git for Windows for Windows v2.45 is the last version to support for Windows 7 and for Windows 8

https://github.com/git-for-windows/git/releases/tag/v2.45.1.windows.1

**Reply by rimrul** [reply]
That's only about Windows 8 (without the `.1`)
